src/api/index.ts

```diff
@@ -7,20 +7,28 @@ export interface Fraction {
 }
 
 export interface CourseItem {
+	id: string;
 	name: string;
 	weight: number;
-	grade?: Grade;
+	grade?: number | Fraction;
 	dueDate?: Date;
 }
```

Comment:
Just realized - should this have an `owner` field at all? @BraidenCutforth

Reply:
Same for the other models that aren't user?

Reply:
Yeah we should have an owner field in the interface but it is unlikely to be used. We also need a timestamp field, https://dynamoosejs.com/guide/Schema/